anova.Rmd

```diff
@@ -342,7 +342,6 @@ p3 = plot_anova(n = 20, mu_a = -1, mu_b = 0, mu_c = 1, sigma = 1)
 Above, there isn't obvious separation between the groups like the first example, but it is still obvious the means are different. Below, there is more noise. Visually it is somewhat hard to tell, but the test still suggests a difference of means. (At an $\alpha$ of 0.05.)
 
 - $\mu_A = -1, \mu_B = 0, \mu_C = 1, \sigma = 2$
-- $n_i = 20$ for each group.
```

Comment:
I do not think this line is necessary because it has been already explained at [this line](https://github.com/daviddalpiaz/appliedstats/blob/c3d743934622cf2595dd7213179d32d9e9ade4f4/anova.Rmd#L295). Also, the other precedent examples do not repeat this variable assignment.